Fuchas/Kernel/Startup/01_luaapi.lua

```diff
@@ -27,72 +20,16 @@ function string.width(...)
 	return unicode.wlen(...)
 end
 
-function string.len(str)
-	if uni then
-		return unicode.len(str)
-	else
-		return _len(str)
-	end
-end
-
-function string.rawlen(str)
-	return _len(str)
-end
-
-function string.sub(str, i, j)
-	if uni then
-		return unicode.sub(str, i, j)
-	else
-		return _sub(str, i, j)
-	end
-end
-
-function string.rawsub(str, i, j)
-	return _sub(str, i, j)
-end
-
-function string.char(...)
-	if uni then
-		return unicode.char(...)
-	else
-		return _char(...)
-	end
-end
-
-function string.reverse(str)
-	if uni then
-		return unicode.reverse(str)
-	else
-		return _reverse(str)
-	end
-end
-
-function string.rawreverse(str)
-	return _reverse(str)
-end
-
-function string.upper(str)
-	if uni then
-		return unicode.upper(str)
-	else
-		return _upper(str)
-	end
-end
-
-function string.lower(str)
-	if uni then
-		return unicode.lower(str)
-	else
-		return _lower(str)
-	end
-end
+string.rawlen = string.len
+string.rawsub = string.sub
+string.rawreverse = reverse
```

Comment:
even of all that could be removed since default unicode now is removed

Reply:
yea i wasn't entirely certain so i left it for compat reasons

Reply:
yea i think it might be used somewhere in io module (not sure), just change that as it's the only place i know of that i used that function